src/qt/glcanvas.cpp

```diff
@@ -490,7 +487,7 @@ bool wxGLCanvas::Create(wxWindow *parent,
 
 bool wxGLCanvas::SwapBuffers()
 {
-    static_cast<QGLWidget *>(m_qtWindow)->swapBuffers();
+    // Not possible
```

Comment:
Wait, what? How does anything gets drawn then?

Reply:
Everything within `paintGL` renders onto a FBO, that is then converted to texture and composited by Qt.

Reply:
"Not possible", as in :

>Triggering a buffer swap just for the QOpenGLWidget is not possible since there is no real, onscreen native surface for it. It is up to the widget stack to manage composition and buffer swaps on the gui thread. ...